telco-examples/mgmt-cluster/aarch64/eib/mgmt-cluster-arm-singlenode.yaml

```diff
@@ -27,57 +27,57 @@ kubernetes:
         createNamespace: true
         installationNamespace: kube-system
       - name: longhorn-crd
-        version: 105.1.0+up1.7.2
+        version: 106.2.0+up1.8.1
         repositoryName: rancher
         targetNamespace: longhorn-system
         createNamespace: true
         installationNamespace: kube-system
       - name: longhorn
-        version: 105.1.0+up1.7.2
+        version: 106.2.0+up1.8.1
         repositoryName: rancher
         targetNamespace: longhorn-system
         createNamespace: true
         installationNamespace: kube-system
       - name: metal3
-        version: 0.9.1
-        repositoryName: suse-edge-upstream
+        version: 303.0.5+up0.11.3
+        repositoryName: suse-edge-charts
         targetNamespace: metal3-system
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: metal3.yaml
       - name: neuvector-crd
-        version: 105.0.0+up2.8.3
+        version: 106.0.0+up2.8.5
         repositoryName: rancher
         targetNamespace: neuvector
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: neuvector.yaml
       - name: neuvector
-        version: 105.0.0+up2.8.3
+        version: 106.0.0+up2.8.5
         repositoryName: rancher
         targetNamespace: neuvector
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: neuvector.yaml
       - name: rancher
-        version: v2.10.1
+        version: 2.11.1
```

Comment:
Sanity check. Did the v gone away?

Reply:
here it's ok, but let me change it in 3.2 and backport it